src/objects.h

```diff
@@ -918,6 +918,15 @@ EXPORT_INLINE void SET_TYPE_DATOBJ(Obj obj, Obj val)
 void SetTypeDatObj(Obj obj, Obj type);
 
 
+/****************************************************************************
+**
+*F  NewKernelBuffer( <size> )  . . . . . . . . . . return a new kernel buffer
+**
+**  TODO: document this
```

Comment:
Do you intend to write this documentation as part of this PR?

Reply:
Oops, that was a mistake. Thanks, I'll fix it ASAP.

Reply:
Fixed now, thanks again.